### PR TITLE
Fix `mindeps`-testing on CI

### PR DESCRIPTION
**Pull request by hendrikmakait**
Closes #xxxx

- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

File: distributed/comm/tests/test_ws.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ async def test_roundtrip(c, s, a, b):
 
 @gen_cluster(client=True, scheduler_kwargs={"protocol": "ws://"})
 async def test_collections(c, s, a, b):
-    da = pytest.importorskip("dask.array")
+    da = pytest.importorskip("dask.array", exc_type=ImportError)
```
Comment (fjetter) [inline distributed/comm/tests/test_ws.py:124]: this is a little odd. why is this necessary?

Comment (hendrikmakait) [reply distributed/comm/tests/test_ws.py:124]: It's either that or adding a `pytest.importorskip("numpy")`. Basically, we raise an ImportError when importing dask.array but don't have the required dependencies over here:

https://github.com/dask/dask/blob/ff2488aec44d641696e0b7aa41ed9e995c710705/dask/array/__init__.py#L266-L273

More context from the `pytest` docs:

> If the module can be imported but raises [ImportError](https://docs.python.org/3/library/exceptions.html#ImportError), pytest will issue a warning to the user, as often users expect the module not to be found (which would raise [ModuleNotFoundError](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError) instead).
>
> This warning can be suppressed by passing exc_type=ImportError explicitly.

(https://docs.pytest.org/en/stable/reference/reference.html#pytest-importorskip)

Comment (phofl) [reply distributed/comm/tests/test_ws.py:124]: I would do a 

pytest.importorskip("numpy")

this is what we do in dask/dask as well

**Review by phofl — APPROVED** [review]
small comment, otherwise lgtm

File: distributed/comm/tests/test_ws.py (hunk 1 above)
Comment (hendrikmakait) [reply distributed/comm/tests/test_ws.py:124]: Works for me.
